src/helpers/index.js

```diff
@@ -30,9 +39,8 @@ function parseOptions(options) {
         }
         adjustments.gridIntensity["device"] = {
           country: device.country,
-          value: parseFloat(
-            averageIntensity.data[device.country?.toUpperCase()]
-          ),
+          // TODO (simon) check that parseFloat can be safely removed here
```

Comment:
question: the types i could find suggest that `averageIntensity.data[device.country?.toUpperCase()]` will always be a number, so `parseFloat` is unnecessary. are there cases where that could be a string?